.openpublishing.redirection.json

```
@@ -4521,6 +4542,31 @@
       "redirect_url": "/dotnet/standard/linq/work-global-namespaces",
       "redirect_document_id": false
     },
+    {
+      "source_path": "docs/visual-basic/programming-guide/concepts/linq/linq-to-xml-axes.md",
+      "redirect_url": "/dotnet/standard/linq/linq-xml-axes-overview",
+      "redirect_document_id": true
+    },
+    {
+      "source_path": "docs/visual-basic/programming-guide/concepts/linq/linq-to-xml-axes-overview.md",
+      "redirect_url": "/dotnet/standard/linq/linq-xml-axes-overview",
+      "redirect_document_id": true
+    },
+    {
+      "source_path": "docs/visual-basic/programming-guide/concepts/linq/serializing-to-an-xmlreader-invoking-xslt.md",
+      "redirect_url": "/dotnet/standard/linq/serialize-xmlreader-invoke-xslt",
+      "redirect_document_id": true
```

if C# version exists and is being redirected, this needs to be false 

```suggestion
      "redirect_document_id": false
```